core-feature-pack/src/main/resources/modules/system/layers/base/org/jboss/as/cli/main/module.xml

```diff
@@ -35,6 +35,25 @@
     </resources>
 
     <dependencies>
+        <!-- JAXP default dependencies. DO NOT REMOVE!!!
+                   These are loaded by the JAXP redirect facility in jboss-modules when this module is used
+                   as a boot module (this is the case for server processes in domain mode). The purpose is
```

Comment:
comment was copy-pasted from server modules, but in any case it is still valid, problem that this change address is booting embedded server. CLI itself works embedded server boot doesn't.